packages/beeq/src/components/drawer/scss/bq-drawer.variables.scss

```diff
@@ -0,0 +1,23 @@
+/* -------------------------------------------------------------------------- */
+/*                   Drawer custom properties                  */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+   * @prop --bq-drawer-width - Default drawer width
+   * @prop --bq-drawer--title-body-gap - Drawer default gap between title and body
+   * @prop --bq-drawer--wrapper-X-padding - Padding on the X-axis for the drawer wrapper
+   * @prop --bq-drawer--wrapper-Y-padding - Padding on the Y-axis for the drawer wrapper
+   * @prop --bq-drawer--footer-top-padding - Padding at the top of the drawer footer
+   * @prop ---bq-drawer--body-padding - Drawer default padding for body slot
+   * @prop --bq-drawer--background-backdrop: Drawer backdrop background color
+   */
+
+  --bq-drawer-width: 320px;
+  --bq-drawer--title-body-gap: theme('spacing.l');
+  --bq-drawer--wrapper-X-padding: theme('spacing.l');
+  --bq-drawer--wrapper-Y-padding: theme('spacing.m');
+  --bq-drawer--footer-top-padding: theme('spacing.m');
+  --bq-drawer--body-padding: theme('spacing.xs');
```

Comment:
Not sure about this, perhaps there's a mistake on the Design, but this how it gonna look if the body have also a padding that's is not aligned with the header padding

![CleanShot 2024-03-13 at 19 32 49@2x](https://github.com/Endava/BEEQ/assets/328492/4b9853da-6499-49cc-a08b-19d859f7a288)